internal/cmdpush/cmdpush.go

```diff
@@ -126,15 +124,15 @@ func (r *Runner) preRunE(_ *cobra.Command, args []string) error {
 	}
 
 	if r.Origin != "" {
-		_, err := parse.ParseArgs(r.ctx, []string{r.Origin, path}, parse.Options{
-			SetOci: func(oci *kptfilev1.Oci) error {
-				r.Push.Origin = remote.NewOciOrigin(oci)
-				return nil
-			},
-		})
+		ref, err := location.ParseReference(
+			r.Origin,
+			location.WithContext(r.ctx),
+			location.WithParsers(location.GitParser, location.OciParser),
```

Comment:
Do we support `push` for git ? if not, then we will have to remove the Git parser so that we can throw error.